fpm/README.md

```diff
@@ -1,14 +1,14 @@
 # PHP FPM on Docker
 
-This is a basic Docker environment for PHP FPM development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php).
+[![Build Status](https://travis-ci.org/akeneo/Dockerfiles.svg?branch=php-7.1)](https://travis-ci.org/akeneo/Dockerfiles/tree/php-7.1)
 
-It is not intended to use directly as it stands, but rather to extend it and create custom development environments.
+This is a basic Docker environment for PHP FPM development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php). **This image does not contain Akeneo PIM**.
 
-It comes with PHP 7.1 (provided by dotdeb.org repository), and some PHP extensions: fpm of course,
-but also apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug (this last one comes deactivated,
-run `phpenmod xdebug` to enable it).
+It is intended to be used [httpd](https://hub.docker.com/_/httpd) with FCGI enable or [nginx](https://hub.docker.com/_/nginx).
 
-The PHP FPM daemon is configured with "docker" as user and group, and listen to the socket "/run/php/php7.1-fpm.sock".
+It comes with Debian 9 (Stretch) and PHP 7.1 (from [Sury repository](https://deb.sury.org/)), and some PHP extensions: fpm of course, but also apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
```

Comment:
`comes deactivated` seems weird but I'm not a very good english speaker :)

I would say:
`this last one is deactivated by default, run `phpenmod xdebug` to enable it`.